inference-engine/tests/ie_test_utils/common_test_utils/file_utils.hpp

```diff
@@ -115,4 +116,67 @@ inline bool directoryExists(const std::string &path) {
     return false;
 }
 
+inline void directoryFileListRecursive(const std::string& name, std::vector<std::string>& file_list) {
```

Comment:
I didn't understand the reason for this structure. Could you please explain?

Reply:
See Patrick's comment https://github.com/openvinotoolkit/openvino/pull/3742#discussion_r576582145